manifest.json

```diff
@@ -15,10 +15,10 @@
   },
   "externally_connectable": {
     "matches": [
-      "http://*/*",
-      "https://*/*"
+      "*://*.b12.io/*"
```

Comment:
great---just add a `NOTE(jrbotos): change this to whichever domain will be sending context change messages to the extension.`